source/shadow.c

```diff
@@ -111,6 +111,20 @@
  */
 #define SHADOW_OP_DELETE_REJECTED_LENGTH     ( SHADOW_OP_DELETE_LENGTH + SHADOW_SUFFIX_REJECTED_LENGTH )
 
+/**
+ * @brief Check if Shadow_MatchTopic has valid parameters.
+ *
+ * @param[in] pTopic Pointer to the topic string.
+ * @param[in] topicLength Length of pTopic.
+ * @param[in] pMessageType Pointer to call-supplied memory for returning the type of the shadow message.
```

Comment:
```suggestion
 * @param[in] pMessageType Pointer to caller-supplied memory for returning the type of the shadow message.
```

Reply:
I agree. This is a block copy from existing documentation in this file. A misguided attempt to reduce PR friction. :-) Done, in all places in shadow.c.